presto-docs/src/main/sphinx/release/release-0.146.rst

```diff
@@ -0,0 +1,9 @@
+=============
+Release 0.146
+=============
+
+* Fix error in map_concat when the second map is empty.
+* Add support for ``VARCHAR(n)`` to Blackhole connector.
```

Comment:
There's only one change, so it seems unnecessary to have a whole section for it.